modules/installation-overview.adoc

```diff
@@ -63,6 +63,11 @@ installer-provisioned infrastructure on the following platforms:
 
 For these clusters, all machines, including the computer that you run the installation process on, must have direct internet access to pull images for platform containers and provide telemetry data to Red Hat.
 
+[IMPORTANT]
+====
+Changing the cloud provider platform after installation is not supported. Likewise, mixing cloud provider components, such as using a persistent storage framework from a differing platform than what the cluster is installed on, is not supported.
```

Comment:
I wonder if it makes sense to omit _Likewise_ and start with _Mixing_?

I also wonder if changing this to a bullet list might make it more scannable?

> After installation, the following changes are unsupported:
> - Cloud provider platform 
> - Mixing component providers, such as using a persistent storage framework from a differing platform than what the cluster is installed on

But written more cogently.